achyut/src/main/java/com/lftechnology/programs/PalindromeProgram.java

```diff
@@ -0,0 +1,52 @@
+package com.lftechnology.programs;
+
+import java.util.Scanner;
+
+public class PalindromeProgram {
+
+    private String number;
+    private Scanner scanner;
+
+    public PalindromeProgram() {
+
+    }
+
+    public void run() {
+        this.initiateRun();
+        System.out.println("Please Enter the Number of Multiplier");
+        scanner = new Scanner(System.in);
+        this.number = scanner.nextLine();
+        Boolean isPalindrome = this.isPalindrome();
+        if (isPalindrome == true) {
+            System.out.println("The number is a palindrome number");
+        } else {
+            System.out.println("The number is not a palindrome number");
+        }
+        this.endRun();
+    }
+
+    public Boolean isPalindrome() {
+        String reverse = "";
+        int length = this.number.length();
+        for (int i = length - 1; i >= 0; i--)
+            reverse = reverse + this.number.charAt(i);
+
+        if (this.number.equals(reverse))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this if-then-else statement by a single return statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1126)